src/js/shared/reducers/SourcesReducer.js

```diff
@@ -0,0 +1,3 @@
+export default function sourcesReducer(state = {}) {
```

Comment:
I'm thinking we should rename this file as sourcesReducer.js. I'm not sure why I had it capitalized...